app/services/hyrax/simple_schema_loader.rb

```diff
@@ -76,7 +76,11 @@ def index_keys
       ##
       # @return [Dry::Types::Type]
       def type
-        collection_type = config['multiple'] ? Valkyrie::Types::Array : Identity
+        collection_type = if config['multiple']
+                            Valkyrie::Types::Array.constructor { |v| Array(v).select(&:present?) }
+                          else
+                            Identity
+                          end
```

Comment:
This is required to remove empty field values passed in from the form.  For example, this changes `creator: [""]` to `creator: []`.